code/datums/keybinding/mob.dm

```diff
@@ -4,7 +4,7 @@
 
 
 /datum/keybinding/mob/face_north
-	hotkey_keys = list("CtrlW", "CtrlNorth")
+	hotkey_keys = list("W", "North")
```

Comment:
```suggestion
	hotkey_keys = list("AltW", "AltNorth")
```

Reply:
yeah this was there as i was tryna fix the issue we discussed